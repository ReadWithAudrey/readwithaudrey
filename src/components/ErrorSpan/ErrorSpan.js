import React from 'react'
import PropTypes from 'prop-types'
import { StyledErrorSpan } from './ErrorSpan.style'

const ErrorSpan = props => {
  return <StyledErrorSpan>{props.children}</StyledErrorSpan>
}

ErrorSpan.propTypes = {
  children: PropTypes.string,
  type: PropTypes.string,
}

export default ErrorSpan
