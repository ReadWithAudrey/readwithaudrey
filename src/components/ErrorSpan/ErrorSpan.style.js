import styled from 'styled-components'

export const HiddenSpan = styled.span.attrs({
  className: 'bg-white pa2 mv2 red w-100 f5 montserrat',
})`
  display: none;
`

export const VisibleSpan = styled.span.attrs({
  className: 'bg-white pa2 mv2 red w-100 f5 montserrat',
})`
  display: flex;
`