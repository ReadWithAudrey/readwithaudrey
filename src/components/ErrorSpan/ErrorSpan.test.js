import React from 'react'
import renderer from 'react-test-renderer'
import ErrorSpan from './ErrorSpan'

describe('Error span component snapshot test', () =>
  it('renders correctly', () => {
    const tree = renderer.create(<ErrorSpan />).toJSON()
    expect(tree).toMatchSnapshot()
  }))
