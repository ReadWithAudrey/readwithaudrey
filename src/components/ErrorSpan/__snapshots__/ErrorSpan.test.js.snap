// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error span component snapshot test renders correctly 1`] = `
<span
  className="sc-bwzfXH bg-white pa2 mv2 red w-100 f5 montserrat jIEWCi"
/>
`;
